Kubecon related changes (#44)
iter8-tools/docs: docs/getting-started/concepts.md

## Iter8 experiment

> The main notion introduced by Iter8 is that of an experiment. An experiment is simply a list of tasks that are executed in a specific sequence.

Reviewer: I think I would prefer

```suggestion
Iter8 introduces the notion of an "experiment". An experiment is simply a list of tasks that are executed in a specific sequence.
```

Author: Fixed.

<p align='center'>
  <img alt-text="Iter8 experiment" src="../../images/authoring.png" width="40%" />
</p>

> Iter8 defines a variety of configurable tasks. Authoring an experiment is as simple as specifying the names of the tasks you wish to include in the experiment, and specifying their parameter values. The following are some examples of tasks provided by Iter8.

Reviewer: ```suggestion
Iter8 provides a variety of configurable tasks. Authoring an experiment is as simple as specifying the names of the tasks and specifying their parameter values. The following are some examples of tasks provided by Iter8.
```

Author: Fixed.
